python/llm/example/transformers/README.md

```diff
@@ -93,4 +94,14 @@ inference:       total time =     xxxx ms
 Inference time (fast forward): xxxx s
 Output:
 {'id': 'cmpl-a0ab2953-e08c-449c-b476-e21ad5bb84b0', 'object': 'text_completion', 'created': 1686557434, 'model': './bigdl_llm_bloom_q4_0.bin', 'choices': [{'text': 'Q: What is CPU? A: central processing unit</s>', 'index': 0, 'logprobs': None, 'finish_reason': None}], 'usage': {'prompt_tokens': None, 'completion_tokens': None, 'total_tokens': None}}
+```
+
+### Model family StarCoder
+```log
+bigdl-llm: mem per token =   313912 bytes
+bigdl-llm:     load time =   479.41 ms
+bigdl-llm:   sample time =    34.29 ms
+bigdl-llm:  predict time =  1365.27 ms / 19.50 ms per token
+bigdl-llm:    total time =  1928.46 ms
```

Comment:
Please hide the detailed time info as others.
![image](https://github.com/intel-analytics/BigDL/assets/54161268/8f92f321-0df6-4f59-abf9-c7b4725ba68c)

And perhaps give an example prediction if it is easy to do :)

Reply:
Done